docs/ops/infrastructure/filestore/privileged-access.rst

```diff
@@ -0,0 +1,72 @@
+##################################
+Privileged access to the filestore
+##################################
+
+Currently, we do not have any way to make containers with privileged
+filesystem access available from JupyterHub.
+
+In order to get privileged access to the filestore, you will need access
+to ``kubectl`` with admin privileges to the cluster you want to work on.
+
+Save the following file as ``copier.yaml``; you may need to edit it to
+point to the correct filestore, and of course if you need multiple
+filestores present (for instance, for copying data between environments)
+then you will need to create multiple Volume/VolumeMount pairs so
+multiple filestores are presented within the container.
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: copier
+      namespace: copier
+    spec:
+      containers:
+      - name: main
+	image: ubuntu:latest
+	args: [ "tail", "-f", "/dev/null" ]
+	volumeMounts:
+	- mountPath: /mnt
+	  name: share
+      volumes:
+      - name: share
+	nfs:
+	  path: /share1
+	  server: 10.13.105.122
+	  # 10.87.86.26 is IDF dev
+	  # 10.22.240.130 is IDF int
+	  # 10.13.105.122 is IDF prod
```

Comment:
Looks like you have tabs, which may break YAML indentation (and at least show up as red in GitHub). Maybe replace all of these with equivalent spaces?